odysseus/odysseus_tree/overlays/rootfs_overlay_tpu/etc/dnsmasq.conf

```diff
@@ -0,0 +1,2 @@
+interface=wlan1     # Use interface wlan1
+dhcp-range=192.168.1.2,192.168.1.254,12h # IP range and lease time
```

Comment:
Is having two DHCP servers in the system going to affect anything? Feels weird especially because these are within the subnet of the router iirc

Reply:
responded in slack